README.md

```diff
@@ -156,6 +156,11 @@ Annotate the branches with GitHub PR numbers: <br/>
 git machete github anno-prs
 ```
 
+Checks out the PR into local branch, if pull request is found on any parent branch, it will add every parent branch up to root branch as seen by git-machete: <br/>
```

Comment:
Hmmm `any parent branch` suggests there might be a few different parent branches for the given PR 🤔

Reply:
Pls clarify the sentence